lib/utils.js

```diff
@@ -25,6 +26,13 @@ class ValidationError extends Error {
     }
 }
 
+function posTag(tokens) {
+    return new POS.Tag(tokens)
+        .initial() // initial dictionary and pattern based tagging
+        .smooth() // further context based smoothing
+        .tags;
+}
```

Comment:
Moved to `american-english.js` in 2c81990. Since schema.org is in English, the entire autoqa is heavily relying on english-specific heuristics. It will be another project to do other languages.

Reply:
It's not just schema.org, right? It's any database schema with meaningful column names. Presumably, a depraved person could have a database schema with Italian column names... Although yeah English is probably the most important language.